### ISSUE TITLE
Allow arrowing around the History window of MSN Messenger / Windows Live Messenger

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#57]
**Reported by mdcurran on 28 Jan 2008 11:51 UTC**
**Blocked by #24**
Currently to read the history window in MSN Messenger and Windows Live Messenger, the user must use the NVDA review commands on the number pad. The reason for this is that the Histroy Window does not have a caret that is accessible to NVDA.
A way this can be fixed is to allow the movement of a virtual caret  provided by NVDA. However this is a separate ticket.

**Comment by michaelDCurran** [comment]
**Modified by mdcurran on 28 Jan 2008 11:52 UTC**

**Comment by michaelDCurran** [comment]
**Modified by mdcurran on 28 Jan 2008 12:02 UTC**

**Comment by michaelDCurran** [comment]
**Modified by jteh on 29 Jan 2008 04:30 UTC**

**Comment by michaelDCurran** [comment]
**Comment by jteh on 11 Feb 2008 04:48 UTC**
Implemented alongside #24 in r1667.
